pymeasure/instruments/instrument.py

```diff
@@ -63,17 +63,36 @@ class Object(object):
 
         self.get = Object()
 
-        # TODO: Determine case basis for the addition of these methods
-        if includeSCPI:
-            # Basic SCPI commands
-            self.status = self.measurement("*STB?",
-                                           """ Returns the status of the instrument """)
-            self.complete = self.measurement("*OPC?",
-                                             """ TODO: Add this doc """)
-
         self.isShutdown = False
         log.info("Initializing %s." % self.name)
 
+    @property
+    def complete(self):
+        """ This property allows synchronization between a controller and a device. The Operation Complete 
+        query places an ASCII character 1 into the device's Output Queue when all pending
+        selected device operations have been finished.
+        """
+        if self.SCPI:
+            return self.adapter.ask("*OPC?").strip()
```

Comment:
It might be nicer to use `self.ask` (which in turn calls `self.adapter.ask`)

Reply:
Ok. I will extend the change also to `id`